docs/en/ingest-management/commands.asciidoc

```diff
@@ -823,8 +823,22 @@ tags, you must unenroll the {agent}, then re-enroll it using new tags.
 Run {agent} without full superuser privileges.
 This option is useful in organizations that limit `root` access on Linux or macOS systems, or `admin` access on Windows systems.
 For details and limitations for running {agent} in this mode, refer to {fleet-guide}/elastic-agent-unprivileged.html[Run {agent} without administrative privileges].
-
++
 Note that changing to `unprivileged` mode is prevented if the agent is currently enrolled in a policy that includes an integration that requires administrative access, such as the {elastic-defend} integration.
++
+[technical preview] To run {agent} without superuser privileges as a pre-existing user or group, for instance under an Active Directory account, you can specify the user or group, and the password to use.
```

Comment:
> [technical preview] To run {agent} without superuser privileges as a pre-existing user or group, for instance under 

This makes me curious why/if https://github.com/elastic/docs?tab=readme-ov-file#using-the-technical-preview-admonition doesn't work in this situation.

Reply:
Good catch. The admonitions work, but the big banner doesn't look right in the context of individual command options. So instead I went with the approach we used in the Elasticsearch docs for a similar problem.